script/v2/keeper-actions/RebalanceScWethV2.s.sol

```diff
@@ -73,6 +73,8 @@ contract RebalanceScWethV2 is Script, scWETHv2Helper {
     // wont have enough wstEth funds for the supply
     uint256 stEthRateTolerance = 0.9995e18;
 
+    bool use0x = false;
```

Comment:
make this public and use it in 0x tests in `RebalanceScWethV2Test` and I suggest to rename it to `useZeroX` as it seems more aligned with other places in the code

Reply:
[Fixed](https://github.com/lindy-labs/sandclock-contracts/pull/151/commits/1ee5088f5d998ccd08d68c8e7db9cefbaeaf2a72)